ebusd/CHANGELOG.md

```
@@ -1,17 +1,71 @@
 # Changelog
 
-# 24.1.1
-## Home Assistant Add-on
+Current Release:
+
+## 25.1
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)